hardware/MySensorsBase.cpp

```diff
@@ -1169,29 +1176,27 @@ void MySensorsBase::UpdateSwitch(const _eSetType vType, const unsigned char Idx,
 		}
 	}
 
-	//Send as Lighting 2
-	tRBUF lcmd;
-	memset(&lcmd, 0, sizeof(RBUF));
-	lcmd.LIGHTING2.packetlength = sizeof(lcmd.LIGHTING2) - 1;
-	lcmd.LIGHTING2.packettype = pTypeLighting2;
-	lcmd.LIGHTING2.subtype = sTypeAC;
-	lcmd.LIGHTING2.id1 = 0;
-	lcmd.LIGHTING2.id2 = 0;
-	lcmd.LIGHTING2.id3 = 0;
-	lcmd.LIGHTING2.id4 = Idx;
-	lcmd.LIGHTING2.unitcode = SubUnit;
+	// LLEMARINEL : #1312  Changed to use as pTypeGeneralSwitch
+	// Send as General Switch :
+	_tGeneralSwitch gswitch;
+	gswitch.subtype = sTypeAC;
```

Comment:
Help! This is the wrong sub type, should not be sTypeAC, have a look in hardwaretypes.h where pTypeGeneralSwitch is defined!
I will create a patch now